gen/calc.py

```diff
@@ -164,6 +178,12 @@ def validate_dcos_overlay_network(dcos_overlay_network):
                 "values are allowed: {}".format(ex))
 
 
+def validate_dcos_remove_dockercfg_enable(dcos_remove_dockercfg_enable):
+    can_be = ['true', 'false']
```

Comment:
nit / not yours: Overall really need to move the "one of" / can be stuff to a common helper all of these call.